api/APIProvider.ts

```diff
@@ -19,10 +19,13 @@ export default class APIProvider {
       let addr = process.env.NEXT_ADDR!;
       let rune = process.env.NEXT_RUNE!;
       let lambda = process.env.NEXT_LAMBDA;
+      let url = "https://api.lnmetrics.info/query";
```

Comment:
```suggestion
      let url = "https://api.lnmetrics.info/query";
```

Can you open a issue to expose this as env variable? this is easy to add but we need to move one